website/client/js/services/userServices.js

```diff
@@ -512,15 +512,15 @@ angular.module('habitrpg')
         },
 
         addWebhook: function (data) {
-          callOpsFunctionAndRequest('addWebhook', 'webhook', "POST", '', data, data.body);
+          callOpsFunctionAndRequest('addWebhook', 'webhook', "POST", '', data);
```

Comment:
@paglias Ok! I was confused that `data.body` was being passed in, considering `callOpsFunctionAndRequest` only takes 5 args.